portalnet/src/metrics/overlay.rs

```diff
@@ -216,12 +244,16 @@ impl OverlayMetrics {
     pub fn get_message_summary(&self) -> String {
         // for every offer you made, how many accepts did you receive
         // for every offer you received, how many accepts did you make
+        let successful_validations = self.validation_count_by_outcome(true);
+        let failed_validations = self.validation_count_by_outcome(false);
         format!(
-            "offers={}/{}, accepts={}/{}",
+            "offers={}/{}, accepts={}/{}, successful_validations={}/{}",
```

Comment:
This log line gets quite long, so maybe we can bend towards shorter labels here, like:
```suggestion
            "offers={}/{}, accepts={}/{}, validations={}/{}",
```